src/app/common/pag-event.ts

```diff
@@ -1,4 +1,22 @@
+/**
+ * Definition of a page for the table
```

Comment:
Probably the name of the interface can be changed, to match the comment?

Reply:
Check if you like more the new comment